src/store/reducers/storage/utils.ts

```diff
@@ -192,13 +196,13 @@ const prepareStorageNodeData = (node: TNodeInfo): PreparedStorageNode => {
 
     const pDisks = node.PDisks?.map((pDisk) => {
         return {
-            ...preparePDiskData(pDisk),
+            ...prepareWhiteboardPDiskData(pDisk),
```

Comment:
Why do we use only whiteboard data and don't use bsc data here?

Reply:
In `prepareStorageNodeData` we prepare data that is received from `viewer/nodes`, it has only whiteboard data (`TPDiskStateInfo `)